forcedotcom/cli
# feat: validate new issues GHA

Adds a new Github action that validates new issues. 
The Action will:
- Post a comment if `sfdx` or `sf` are older than latest
- Post a comment if the full `--verbose` version output was not included
- Add the `more information needed` label

Here is an example of all 3 cases: https://github.com/iowillhoit/gha-sandbox/issues/22

This PR also updates ISSUE_TEMPLATE

Testing Instructions:
- Create issues here: https://github.com/iowillhoit/gha-sandbox/issues
- Versions check looks for `@salesforce/cli@x.x.x` and `sfdx-cli/x.x.x`
- Verbose check looks for `pluginVersions` or `Plugin Version:` (to support json and stdout)
